build-system/pr-check/build-targets.js

```diff
@@ -172,12 +196,21 @@ const targetMatchers = {
       })
     );
   },
+  [Targets.LINT]: (file) => {
```

Comment:
Nit: `(file) => lintFiles.includes(file)`

Reply:
Had to add a clause to omit owners files from lint / presubmit, so I left the long-form predicate for readability.